src/routes/subscriptions.rs

```diff
@@ -2,7 +2,7 @@ use std::sync::Arc;
 
 use axum::{
     extract::{Form, State},
-    http::StatusCode,
+    http,
```

Comment:
Switched to just `http`, because `reqwest` has a child `StatusCode` enum and I didnt want it to clash